model/utils/data_preprocess.py

```diff
@@ -61,7 +62,7 @@ def encode_onehot(df):
     return df
 
 
-def preprocess_data(df, df_test=None):
+def preprocess_data(df: pd.DataFrame, df_test=None) -> pd.DataFrame:
```

Comment:
```suggestion
def preprocess_data(df: pd.DataFrame, df_test: Optional[pd.DataFrame] = None) -> pd.DataFrame:
```
I think it is `pd.DataFrame` type but I'm not sure.